.travis.yml

```diff
@@ -3,7 +3,7 @@ install:
   - wget https://s3.amazonaws.com/rebar3/rebar3 && chmod +x rebar3
 language: erlang
 otp_release:
-  - 20.3
+  - 21.1
```

Comment:
We don't support upgrade to 21.1 yet (several system wide compatibility issues)

Reply:
I'm now back to 20.3.